core/src/main/scala/libretto/scaletto/impl/FreeScaletto.scala

```diff
@@ -457,21 +457,20 @@ object FreeScaletto extends FreeScaletto with Scaletto {
       id
   }
 
-  implicit val csmc: ClosedSymmetricMonoidalCategory[-⚬, |*|, One, =⚬] =
-    new ClosedSymmetricMonoidalCategory[-⚬, |*|, One, =⚬] {
-      override def andThen[A, B, C](f: A -⚬ B, g: B -⚬ C): A -⚬ C                              = FreeScaletto.this.andThen(f, g)
-      override def id[A]: A -⚬ A                                                               = FreeScaletto.this.id[A]
-      override def par[A1, A2, B1, B2](f1: A1 -⚬ B1, f2: A2 -⚬ B2): (A1 |*| A2) -⚬ (B1 |*| B2) = FreeScaletto.this.par(f1, f2)
-      override def assocLR[A, B, C]: ((A |*| B) |*| C) -⚬ (A |*| (B |*| C))                    = FreeScaletto.this.assocLR[A, B, C]
-      override def assocRL[A, B, C]: (A |*| (B |*| C)) -⚬ ((A |*| B) |*| C)                    = FreeScaletto.this.assocRL[A, B, C]
-      override def swap[A, B]: (A |*| B) -⚬ (B |*| A)                                          = FreeScaletto.this.swap[A, B]
-      override def elimFst[A]: (One |*| A) -⚬ A                                                = FreeScaletto.this.elimFst[A]
-      override def elimSnd[A]: (A |*| One) -⚬ A                                                = FreeScaletto.this.elimSnd[A]
-      override def introFst[A]: A -⚬ (One |*| A)                                               = FreeScaletto.this.introFst[A]
-      override def introSnd[A]: A -⚬ (A |*| One)                                               = FreeScaletto.this.introSnd[A]
-      override def curry[A, B, C](f: (A |*| B) -⚬ C): A -⚬ (B =⚬ C)                            = FreeScaletto.this.curry(f)
-      override def eval[A, B]: ((A =⚬ B) |*| A) -⚬ B                                           = FreeScaletto.this.eval[A, B]
-    }
+  given ℭ: ClosedSymmetricMonoidalCategory[-⚬, |*|, One, =⚬] with {
```

Comment:
Not sure you like Unicode… The name is cryptic anyway, but now it looks more funky (and is shorter on call side). :smile:

Reply:
As this is not a regular user would face, I don't care much.